index.html

```diff
@@ -0,0 +1,202 @@
+            id="menu-icon"
+          />
+        </div>
+        <div class="hero-container">
+          <div class="hero-image">
+            <h1>San Diego Comic-Con International</h1>
+          </div>
+          <div class="message">
+            <p>
+              A joyful celebration believing in the value of openness and
+              sharing, creating a positive change with people from over 80
+              countries is taking place in October, in Korea.
+            </p>
+          </div>
+          <div class="location">
+            <h2>2015.10.15(THU) ~ 16(FRI)</h2>
+            <p>@National Museum of Korea, Art Center Nabi, and more</p>
+          </div>
+        </div>
+      </section>
+      <section id="main-program">
+        <div class="main-program-title">
+          <h2>Main Program</h2>
+          <span></span>
+        </div>
+        <div class="cards-container">
+          <article>
+            <img
+              src="./images/icons/program_icon_01.png"
+              alt="program_icon_01"
+            />
+            <h3>Lecture</h3>
+            <p>
+              Listen to the speakers from various countries about the messages
+              of sharing and opening.
+            </p>
+          </article>
+
+          <article>
+            <img
+              src="./images/icons/program_icon_02.png"
+              alt="program_icon_02"
+            />
+            <h3>CC Exhibition</h3>
+            <p>
+              Appreciate various creations applying CC license of artists,
+              organized from Art Center Nabi.
+            </p>
+          </article>
+
+          <article>
+            <img
+              src="./images/icons/program_icon_03.png"
+              alt="program_icon_03"
+            />
+            <h3>Forum</h3>
+            <p>
+              Have the time to share your thoughts and opinions with experts for
+              each topic.
+            </p>
+          </article>
+
+          <article>
+            <img
+              src="./images/icons/program_icon_04.png"
+              alt="program_icon_04"
+            />
+            <h3>Workshop</h3>
+            <p>
+              Try creating your own work using open source license rather than
+              just watching.
+            </p>
+          </article>
+
+          <article>
+            <img
+              src="./images/icons/program_icon_05.png"
+              alt="program_icon_05"
+            />
+            <h3>CC Ignite</h3>
+            <p>
+              Get opportunities to network with CC affiliates around the world,
+              also after the summit.
+            </p>
+          </article>
+        </div>
+        <div class="button">
+          <button type="button">Join CC Global Summit 2015</button>
+        </div>
+      </section>
+      <section id="featured-speakers"></section>
+      <section id="partners-section">
+        <div class="partners-title">
+          <h2>Partners</h2>
```

Comment:
Good job! but please make sure the partners and the footer section are visible on the mobile screen.

![image](https://user-images.githubusercontent.com/34779928/197276147-6b5f2fc4-3d4a-40eb-bde8-6eee2e4779a8.png)

Reply:
@aimalamiri i make just like the design on behance the previous reviewer told me to be like the design exactly

Reply:
@belalgamal46, actually I was talking about the footer, because when you scroll down in the mobile screen you're not able to see it.

Reply:
@aimalamiri yeah because that is how it looks in the design on behance
![image](https://user-images.githubusercontent.com/32738452/197336531-7cdb5ebb-fc66-4087-a669-1ed2c5a42b6b.png)